Helper/Asset/ProcessHelper.php

```diff
@@ -46,10 +46,10 @@ public function process(Request $request, string $processor, string $assetHash,
         @trigger_error("ProcessHelper::process is deprecated use the ems_asset twig filter to generate the route", E_USER_DEPRECATED);
 
         if ($configHash) {
-            return $this->processor->fromCache($request, $processor, $assetHash, $configHash);
+            return $this->processor->getResponse($request, $processor, $assetHash, $configHash);
         }
 
-        return  $this->processor->createResponse($request, $processor, $assetHash, $this->getOptions($processor));
+        return new Response('Deprecated use ems_asset twig filter', 500);
```

Comment:
@theus77 mathieu you aggree with this? They processor does not support GetResponse without a configHash.

fromCache and createResponse remove in this commit:
https://github.com/ems-project/EMSCommonBundle/commit/af1d6c7e1172835fca677eb14dab3c8b159aca57